docs/PC-8800.md

```diff
@@ -44,40 +25,171 @@ At this link you can find information on the BIOS files of the emulator:
 | N88EXT2.ROM | B7C8BCEA219B77D9CC3EE0EFAFE343CC307425D1 | AF2B6EFA | 8 KB |
 | N88EXT3.ROM | EFCE0B51CAB9F0DA6CF68507757F1245A2867A72 | 7713C519 | 8 KB |
 
-If you add these 2 files too, you will have the translation of the menus in kanji characters:
+Adding the next 2 files :
+
+Menus (Quasi88 only) and Japanese games will display the Japanese kanji characters. [Quasi88, lr-quasi88]
+
+Not adding the next 2 files :
+
+Menus will be in English (Quasi88 only) and Japanese games will **not** display the Japanese kanji characters. [Quasi88, lr-quasi88]
 
 | File | SHA1 | CRC32 | Size |
 | :--: | :--: | :--: | :--: |
 | N88KNJ1.ROM | 82E11A177AF6A5091DD67F50A2F4BAFDA84D6556 | 6178BD43 | 128 KB |
 | N88KNJ2.ROM | 7E6591CD465CBB35D6D3446C5A83B46D30FAFE95 | 154803CC | 128 KB |
 
-Put the bios files in lowercase otherwise the emulator won't find them, and result in black screen.
```

Comment:
Why was this deleted - is the file casing not important ?

Reply:
Quasi88 works with uppercase files. Lr-quasi88 looks first for uppercase files then lowercase files. Discovered this in while reading the lr-quasi88 logs.